backend/src/main/java/com/jobwait/persistence/PostgresController.java

```diff
@@ -133,12 +134,12 @@ INSERT INTO answers (%s)
     }
 
     @Override
-    public User createUserFromAuthId(String authHash) {
+    public User createUser(Tuple<String, String> userIDAndHash) {
         try {
             Connection connection = getConnection();
 
             PreparedStatement statement = connection
-                    .prepareStatement("INSERT INTO users(authhash) VALUES (?) RETURNING *");
+                    .prepareStatement("INSERT INTO users(id, authhash) VALUES (?, ?) RETURNING *");
             new PostgresUserAdapter().statementSetPlaceholders(statement, new User(null, authHash));
```

Comment:
This also doesn't appear to be using the `userIDAndHash` parameter, I guess this is part of the 30% left to be done

Reply:
Correct, please see comment: https://github.com/pecasdev/jobwait/pull/24#discussion_r1763564387